conf/application.conf

```diff
@@ -66,13 +66,13 @@ microservice {
         port = 9022
         protocol = http
       }
-    }
 
-    business-matching {
-      host = localhost
-      port = 9756
-      protocol = http
-      startUrl = "/register-for-cross-border-arrangements"
+      business-matching {
+        host = localhost
+        port = 9756
+        protocol = http
+        startUrl = "/register-for-cross-border-arrangements"
```

Comment:
@iservport Please check the URL and port number

Reply:
Please see the above comment for the URLS